diffcp/cone_program.py

```diff
@@ -165,12 +181,30 @@ def derivative(dA, db, dc, **kwargs):
             [-dA, None, np.expand_dims(db, -1)],
             [-np.expand_dims(dc, -1).T, -np.expand_dims(db, -1).T, None]
         ])
-        # can ignore w since w = 1
         rhs = dQ @ pi_z
         if np.allclose(rhs, 0):
             dz = np.zeros(rhs.size)
-        else:
+        elif mode == "dense":
```

Comment:
We can remove this iterative refinement stuff for now. Too complicated and it'll probably go to C++ anyways and also we don't have sparse implemented in c++ yet anyways.

What do you guys think?

Reply:
Fine with me, can remove it if you agree @akshayka?

Reply:
Sure, that's fine by me.